src/components/calculator.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Linters are useful for ensuring we write clean code so please do not disable them.